brendenwest/cascade
# Daily Rides API and Site Linking

## Description
Resolves Issue #38 

This PR accomplishes api fetching and linking to external https pages.
It renders fetched data in Flatlist format, where elements are loaded up as the user scrolls the list down. Item filtering is not implemented as part of this PR.

## To Test
1. pull the branch, run Metro, run the app
2. Click on Daiy Rides in the left hand side drawer menu
3. Observe api information loading in Flatlist format 
*NOTE* if you are not testing on a phone but with an emulator, make sure to use your computer IP address and not localhost as described [here](https://stackoverflow.com/questions/38418998/react-native-fetch-network-request-failed) 
4. Click on any event and observe the link opening in the browser
5. Scroll the list down and observe additional rides rendering
6. Ensure no errors or warnings pop up

This PR does not include linking to individual ride pages within the app, filtering of daily rides based on date or stacking on the Home page. These features are to be addressed in subsequent PRs
